smithy-syntax/src/main/java/software/amazon/smithy/syntax/BracketFormatter.java

```diff
@@ -103,15 +104,38 @@ BracketFormatter forceLineBreaksIfNotEmpty() {
         return this;
     }
 
+    // Don't force line breaks and don't indent inside brackets
+    BracketFormatter inline(boolean inline) {
```

Comment:
Not sure I understand what this setter does. Is this really "forceInline"? If so, maybe it should set forceLineBreaks to false, and forceLineBreaks when set to true sets forceInline to false. That way you can remove the other conditionals around it, and the last one set wins.